plugins/in_dummy/in_dummy.c

```diff
@@ -398,7 +402,12 @@ static struct flb_config_map config_map[] = {
    {
     FLB_CONFIG_MAP_INT, "rate", "1",
     0, FLB_TRUE, offsetof(struct flb_dummy, rate),
-    "set a number of events per second."
+    "set a number of events per time interval."
+   },
+   {
+    FLB_CONFIG_MAP_INT, "time_interval", "1",
```

Comment:
How about setting the property name to `Interval_Sec` ?
Many plugins support `Interval_Sec` to set polling interval in second. It is consistent.

```
$ find . -name "*.c"|xargs grep -i \"interval_sec\"
./in_winevtlog/in_winevtlog.c:      FLB_CONFIG_MAP_TIME, "interval_sec", "1s",
./in_random/random.c:      FLB_CONFIG_MAP_INT, "interval_sec", DEFAULT_INTERVAL_SEC,
./in_kubernetes_events/kubernetes_events.c:      FLB_CONFIG_MAP_INT, "interval_sec", DEFAULT_INTERVAL_SEC,
./in_proc/in_proc.c:      FLB_CONFIG_MAP_INT, "interval_sec", DEFAULT_INTERVAL_SEC,
./in_winlog/in_winlog.c:      FLB_CONFIG_MAP_TIME, "interval_sec", "1s",
./in_mem/mem.c:      FLB_CONFIG_MAP_INT, "interval_sec", DEFAULT_INTERVAL_SEC,
./in_calyptia_fleet/in_calyptia_fleet.c:      FLB_CONFIG_MAP_INT, "interval_sec", DEFAULT_INTERVAL_SEC,
./in_winstat/winstat.c:      FLB_CONFIG_MAP_TIME, "interval_sec", "1s",
./in_cpu/cpu.c:      FLB_CONFIG_MAP_INT, "interval_sec", DEFAULT_INTERVAL_SEC,
./in_event_type/event_type.c:      FLB_CONFIG_MAP_INT, "interval_sec", DEFAULT_INTERVAL_SEC,
./in_docker/docker.c:      FLB_CONFIG_MAP_INT, "interval_sec", DEFAULT_INTERVAL_SEC,
./in_exec/in_exec.c:      FLB_CONFIG_MAP_INT, "interval_sec", DEFAULT_INTERVAL_SEC,
./in_thermal/in_thermal.c:      FLB_CONFIG_MAP_INT, "interval_sec", DEFAULT_INTERVAL_SEC,
./in_netif/in_netif.c:      FLB_CONFIG_MAP_INT, "interval_sec", DEFAULT_INTERVAL_SEC,
./in_disk/in_disk.c:      FLB_CONFIG_MAP_INT, "interval_sec", DEFAULT_INTERVAL_SEC,
./in_exec_wasi/in_exec_wasi.c:      FLB_CONFIG_MAP_INT, "interval_sec", DEFAULT_INTERVAL_SEC,
./in_head/in_head.c:      FLB_CONFIG_MAP_INT, "interval_sec", DEFAULT_INTERVAL_SEC,
./in_health/health.c:      FLB_CONFIG_MAP_INT, "interval_sec", DEFAULT_INTERVAL_SEC,
```

Reply:
I agree with the comments and settings proposal. Removed `time_interval` and added `interval_sec/nsec` that overrides the `rate` setting when set.

Reply:
Re-requesting review from @braydonk and @nokute78.